src/_modules/common_values/outputs_network.tf

```diff
@@ -0,0 +1,136 @@
+    }
+    firmaconio_selfcare = {
+      name                = "firmaconio.selfcare.pagopa.it"
+      resource_group_name = "${local.project_weu_legacy}-rg-external"
+    }
+    io_web = {
+      name                = "ioapp.it"
+      resource_group_name = "${local.project_weu_legacy}-rg-external"
+    }
+  }
+}
+
+output "private_dns_zones" {
+  description = <<EOF
+  WHAT: All private dns zones
+  EOF
+  value = {
+    io_papopa = {
+      name                = "internal.io.pagopa.it"
+      resource_group_name = "${local.project_weu_legacy}-rg-internal"
+    }
+    redis = {
+      name                = "privatelink.redis.cache.windows.net"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    postgres = {
+      name                = "privatelink.postgres.database.azure.com"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    mysql = {
+      name                = "privatelink.mysql.database.azure.com"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    azurecr = {
+      name                = "privatelink.azurecr.io"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    mongo = {
+      name                = "privatelink.mongo.cosmos.azure.com"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    servicebus = {
+      name                = "privatelink.servicebus.windows.net"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    documents = {
+      name                = "privatelink.documents.azure.com"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    blob_core = {
+      name                = "privatelink.blob.core.windows.net"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    queue_core = {
+      name                = "privatelink.queue.core.windows.net"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    table_core = {
+      name                = "privatelink.table.core.windows.net"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    azurewebsites = {
+      name                = "privatelink.azurewebsites.net"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    search = {
+      name                = "privatelink.search.windows.net"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    azure_api = {
+      name                = "azure-api.net"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    management_azure_api = {
+      name                = "management.azure-api.net"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+    scm_azure_api = {
+      name                = "scm.azure-api.net"
+      resource_group_name = "${local.project_weu_legacy}-rg-common"
+    }
+  }
+}
```

Comment:
Can we get this information from the core state with the data.terraform_remote_state.core datasource?
Probably some of them must be gotten from the common state

Reply:
I tried, but from that state I didn't find any information about private dns zones

Reply:
I agree with this proposal. Moreover, as a developer I would prefer to see outputs grouped by service type as it looks simpler. Also naming, what do you think to use the actual service name instead of the private dns zone name (developers don't care)? i.e. `azurewebsites` becomes `app_service` or `app`, `documents` becomes `cosmos` or `cosno`, etc.